src/DisplayBoard.cpp

```diff
@@ -7,19 +7,27 @@ namespace
     const char INTERSECTION = '+';
     const char PADDING = ' ';
 
-
     std::string drawPadding(const unsigned int width,
                             const char sign = PADDING);
 
-    std::string drawEndLine(const unsigned int widthRows, const unsigned int width = 0);
+    std::string drawEndLine(const unsigned int widthRows,
+                            const unsigned int width = 0);
+
     std::string drawEmptyLine(const unsigned int width,
                               const char sign = PADDING);
 
     std::string drawColumns(const unsigned int maxElementsInRows,
                             const unsigned int sizeCols,
-                            const unsigned int heightCol);
+                            const unsigned int heightCol,
+                            const std::vector<std::string>& stringClues);
```

Comment:
i think there was a disscussion about putting space before and after & in argument declaration